frontend/src/Components/Link/ClipboardButton.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import FormInputButton from 'Components/Form/FormInputButton';
+import Icon from 'Components/Icon';
+import { icons, kinds } from 'Helpers/Props';
+import { ButtonProps } from './Button';
+import styles from './ClipboardButton.css';
+
+export interface ClipboardButtonProps extends Omit<ButtonProps, 'children'> {
+  value: string;
+}
+
+export type ClipboardState = 'success' | 'error' | null;
+
+export default function ClipboardButton({
+  id,
+  value,
+  className = styles.button,
+  ...otherProps
+}: ClipboardButtonProps) {
+  const [state, setState] = useState<ClipboardState>(null);
+
+  useEffect(() => {
+    const handler =
+      state &&
+      setTimeout(() => {
+        setState(null);
+      }, 3000);
+
+    return () => {
+      if (handler) clearTimeout(handler);
+    };
+  }, [state]);
```

Comment:
Creating a `useDebounce` hook might be useful if this is a recurrent pattern in the app